tests/third_party/cupy/indexing_tests/test_indexing.py

```diff
@@ -99,7 +99,6 @@ def test_diagonal(self, xp, dtype):
         a = testing.shaped_arange((3, 4, 5), xp, dtype)
         return a.diagonal(1, 2, 0)
 
-    @pytest.mark.usefixtures("allow_fall_back_on_numpy")
     @testing.for_all_dtypes()
```

Comment:
can we add `third_party/cupy/indexing_tests/test_indexing.py` scope to public CI?